gef.py

```diff
@@ -411,29 +413,29 @@ def inner_f(*args, **kwargs):
     return wrapper
 
 
-def only_if_events_supported(event_type):
+def only_if_events_supported(event_type) -> Callable:
     """Checks if GDB supports events without crashing."""
-    def wrap(f):
-        def wrapped_f(*args, **kwargs):
+    def wrap(f: Callable) -> Callable:
+        def wrapped_f(*args: Tuple, **kwargs: Dict) -> Callable:
```

Comment:
We can't know that this returns `Callable` because it returns whatever the wrapped function returns.